pretext/Lists/ListMethods.ptx

```diff
@@ -188,23 +155,89 @@ print(mylist)
+
+
+  <program xml:id="ch4_sec10_p1" interactive="activecode" language="python">
     <input>
+# Let's create a list and fill it with some values
 mylist = []
 mylist.append(5)
 mylist.append(27)
 mylist.append(3)
 mylist.append(12)
 print(mylist)
+        </input>
+  </program>
+
+<p> Another way to add items: </p>
+
+  <program xml:id="ch4_sec10_p2" interactive="activecode" language="python">
+    <input>
+mylist = [5, 27, 3, 12]
+mylist.insert(1, 12)        # Insert 12 at pos 1, shift other items up
+print(mylist)
+        </input>
+  </program>
+
+<p> Some return type methods: </p>
 
-mylist = mylist.sort()   #probably an error
+  <program xml:id="ch4_sec10_p3" interactive="activecode" language="python">
+    <input>
+mylist = [5, 27, 3, 12]
+print(mylist.count(12))     # How many times is 12 in mylist?
+print(mylist.index(3))      # Find index of first 3 in mylist
+print(mylist)              # mylist is unchanged as count and index are return type methods
+    </input>
+  </program>
+
+<p> Some mutator methods: </p>
+
+  <program xml:id="ch4_sec10_p4" interactive="activecode" language="python">
+    <input>
+mylist = [5, 27, 3, 12]
+mylist.reverse()
 print(mylist)
+
+mylist.sort()
+print(mylist)
+
+mylist.remove(5)            # Removes the element on the fifth index (first 12) of the list
+print(mylist)
+
+my_second_list = mylist.append(5)   # We shouldn't assign the result of a mutator method to a variable!
+print(my_second_list)               # It will be None
+      </input>
+  </program>
+
+<p> Some hybrid methods: </p>
+
+
+  <program xml:id="ch4_sec10_p5" interactive="activecode" language="python">
+    <input>
+mylist = [5, 27, 3, 12]
+
+lastitem = mylist.pop()     # Removes and returns the last item of the list
+print(lastitem)
+print(mylist)
+
+seconditem = mylist.pop(1)   # Removes and returns the item at index 1
+print(seconditem)
+
+
         </input>
   </program>
+  <p>Notice that, there are two ways to use the <c>pop</c> method.  The first, with no parameter, will remove and return the
+            last item of the list.  If you provide a parameter for the position, <c>pop</c> will remove and return the
+            item at that position.  Either way the list is changed.</p>
+ 
+  <p>Details for these and others
+            can be found in the <url href="http://docs.python.org/py3k/library/stdtypes.html#sequence-types-str-bytes-bytearray-list-tuple-range" visual="http://docs.python.org/py3k/library/stdtypes.html#sequence-types-str-bytes-bytearray-list-tuple-range">Python Documentation</url>.</p>
+  <p>It is important to remember that methods like <c>append</c>, <c>sort</c>,
+            and <c>reverse</c> all return <c>None</c>.  This means that re-assigning <c>mylist</c> to the result of sorting <c>mylist</c> will result in losing the entire list.  Calls like these will likely never appear as part of an assignment statement (see line 8 below).</p>
```

Comment:
I feel like this could be better explained as an example.

Reply:
I placed this sentence next to its example